service CurtainService {
    rpc open(StringRequest) returns (StringResponse) {}
    rpc close(StringRequest) returns (StringResponse) {}
    rpc adjustHeightAndWidth(HeightAndWidth) returns (StringResponse) {}
}

message StringRequest {
    string stringRequestValue = 1;
}

message BooleanRequest {
    bool booleanRequestValue = 1;
}

message StringResponse {
    string stringResponseValue = 1;
}

message HeightAndWidth {
    float height = 1;
    float width = 2;
}

